docs/docker-stack/docker-examples/restricted/restricted_environments.sh

```diff
@@ -22,28 +22,29 @@ AIRFLOW_SOURCES="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../../" && pwd)"
 cd "${AIRFLOW_SOURCES}"
 
 # [START download]
+export AIRFLOW_VERSION="2.3.0.dev0"
 rm docker-context-files/*.whl docker-context-files/*.tar.gz docker-context-files/*.txt || true
 
 curl -Lo "docker-context-files/constraints-3.7.txt" \
-    https://raw.githubusercontent.com/apache/airflow/constraints-2.3.0.dev0/constraints-3.7.txt
+    "https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-3.7.txt"
 
 # For Airflow pre 2.1 you need to use PIP 20.2.4 to install/download Airflow packages.
 pip install pip==20.2.4
```

Comment:
Is this still needed? Or maybe we need to parse `AIRFLOW_VERSION` to decide what version of pip to use?

Reply:
Yeah. Lets' remove it It used to be needed at the beginning when I wanted to show example of using previous PIP version but that need is long gone.